ui/ducks/send/send.duck.js

```diff
@@ -1,382 +0,0 @@
-        tokenBalance: null,
-        from: '',
-        to: '',
-        amount: '0x0',
-        memo: '',
-        errors: {},
-        maxModeOn: false,
-        editingTransactionId: null,
-        toNickname: '',
-      };
-    case GAS_LOADING_STARTED:
-      return {
-        ...state,
-        gasIsLoading: true,
-      };
-
-    case GAS_LOADING_FINISHED:
-      return {
-        ...state,
-        gasIsLoading: false,
-      };
-    default:
-      return state;
-  }
-}
-
-// Action Creators
-export function openToDropdown() {
-  return { type: OPEN_TO_DROPDOWN };
-}
-
-export function closeToDropdown() {
-  return { type: CLOSE_TO_DROPDOWN };
-}
-
-export function showGasButtonGroup() {
-  return { type: SHOW_GAS_BUTTON_GROUP };
-}
-
-export function hideGasButtonGroup() {
-  return { type: HIDE_GAS_BUTTON_GROUP };
-}
-
-export function updateSendErrors(errorObject) {
-  return {
-    type: UPDATE_SEND_ERRORS,
-    value: errorObject,
-  };
-}
-
-export function resetSendState() {
-  return { type: RESET_SEND_STATE };
-}
-
-export function setGasLimit(gasLimit) {
-  return {
-    type: UPDATE_GAS_LIMIT,
-    value: gasLimit,
-  };
-}
-
-export function setGasPrice(gasPrice) {
-  return {
-    type: UPDATE_GAS_PRICE,
-    value: gasPrice,
-  };
-}
-
-export function setGasTotal(gasTotal) {
-  return {
-    type: UPDATE_GAS_TOTAL,
-    value: gasTotal,
-  };
-}
-
-export function updateGasData({
-  gasPrice,
-  blockGasLimit,
-  selectedAddress,
-  sendToken,
-  to,
-  value,
-  data,
-}) {
-  return (dispatch) => {
-    dispatch(gasLoadingStarted());
-    return estimateGasForSend({
-      estimateGasMethod: estimateGas,
-      blockGasLimit,
-      selectedAddress,
-      sendToken,
-      to,
-      value,
-      estimateGasPrice: gasPrice,
-      data,
-    })
-      .then((gas) => {
-        dispatch(setGasLimit(gas));
-        dispatch(setCustomGasLimit(gas));
-        dispatch(updateSendErrors({ gasLoadingError: null }));
-        dispatch(gasLoadingFinished());
-      })
-      .catch((err) => {
-        log.error(err);
-        dispatch(updateSendErrors({ gasLoadingError: 'gasLoadingError' }));
-        dispatch(gasLoadingFinished());
-      });
-  };
-}
-
-export function gasLoadingStarted() {
-  return {
-    type: GAS_LOADING_STARTED,
-  };
-}
-
-export function gasLoadingFinished() {
-  return {
-    type: GAS_LOADING_FINISHED,
-  };
-}
-
-export function updateSendTokenBalance({ sendToken, tokenContract, address }) {
-  return (dispatch) => {
-    const tokenBalancePromise = tokenContract
-      ? tokenContract.balanceOf(address)
-      : Promise.resolve();
-    return tokenBalancePromise
-      .then((usersToken) => {
-        if (usersToken) {
-          const newTokenBalance = calcTokenBalance({ sendToken, usersToken });
-          dispatch(setSendTokenBalance(newTokenBalance));
-        }
-      })
-      .catch((err) => {
-        log.error(err);
-        updateSendErrors({ tokenBalance: 'tokenBalanceError' });
```

Comment:
Huh, apparently this error wasn't valid :thinking: No localized message with the key `tokenBalanceError`.